services/traction/api/db/models/v1/contact.py

```diff
@@ -83,6 +88,84 @@ class Contact(BaseModel, table=True):
         )
     )
 
+    @classmethod
```

Comment:
👍 like this, let's keep doing this, simple gets/read functions in the "table" class. maybe we can add an optional bool for returning None or throwing the exception.